plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled, AdhocMetric, getNumberFormatter } from '@superset-ui/core';
+// @ts-ignore
+import PivotTable from '@superset-ui/react-pivottable/PivotTable';
+// @ts-ignore
+import { sortAs, aggregatorTemplates } from '@superset-ui/react-pivottable/Utilities';
+import '@superset-ui/react-pivottable/pivottable.css';
+import { PivotTableProps, PivotTableStylesProps } from './types';
+
+const Styles = styled.div<PivotTableStylesProps>`
+  padding: ${({ theme }) => theme.gridUnit * 4}px;
+  height: ${({ height }) => height}px;
+  width: ${({ width }) => width}px;
+  overflow-y: scroll;
+  }
+`;
+
+const clickCellCallback = (
+  e: MouseEvent,
+  value: number,
+  filters: Record<string, any>,
+  pivotData: Record<string, any>,
+) => {
+  console.log('CELL CLICKED', { e, value, filters, pivotData });
+};
+
+const clickColumnHeaderCallback = (
+  e: MouseEvent,
+  value: string,
+  filters: Record<string, any>,
+  pivotData: Record<string, any>,
+  isSubtotal: boolean,
+  isGrandTotal: boolean,
+) => {
+  console.log('COL CLICKED', { e, value, filters, pivotData, isSubtotal, isGrandTotal });
+};
+
+const clickRowHeaderCallback = (
+  e: MouseEvent,
+  value: string,
+  filters: Record<string, any>,
+  pivotData: Record<string, any>,
+  isSubtotal: boolean,
+  isGrandTotal: boolean,
+) => {
+  console.log('ROW CLICKED', { e, value, filters, pivotData, isSubtotal, isGrandTotal });
+};
```

Comment:
Maybe we could comment out the `console.log`s here until we implement filters/other events.

Reply:
Done. I replaced `console.log`s with `TODO` comments